wsf_scraping/pipelines.py

```diff
@@ -32,7 +33,12 @@ def __init__(self):
         spiders = spider_loader.list()
 
         for spider_name in spiders:
-            folder_path = os.path.join('./', 'results', 'pdfs', spider_name)
+            folder_path = os.path.join(
+                os.path.curdir,
```

Comment:
I don’t think you need this – it’s assumed to be relative to the current directory if not.